src/DataFixtures/Faker/Provider/UserProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures\Faker\Provider;
+
+use App\Domain\SkillSetDomain;
+
+class UserProvider
+{
+    private static int $count = 990000;
+
+    private SkillSetDomain $skillSetDomain;
+
+    public function __construct(SkillSetDomain $skillSetDomain)
+    {
+        $this->skillSetDomain = $skillSetDomain;
+    }
+
+    public static function identificationNumber(): string
```

Comment:
This method is not used in any yaml file, it can be safely removed

Reply:
done